plugins/cloud-azure/src/main/java/org/elasticsearch/cloud/azure/storage/AzureStorageServiceImpl.java

```diff
@@ -175,12 +175,14 @@ public OutputStream getOutputStream(String container, String blob) throws URISyn
 
         CloudBlobContainer blob_container = client.getContainerReference(container);
```

Comment:
Since we are modifying this code, can we change `blob_container` -> `blobContainer`?